<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebDialogPCAItemsSettings</class>
 <widget class="QWidget" name="WebDialogPCAItemsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Shape</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_shape"/>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_size"/>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="comboBox_color"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
